src/Reflection/Exception/ClassDoesNotExist.php

```diff
@@ -4,8 +4,14 @@
 
 namespace Roave\BetterReflection\Reflection\Exception;
 
+use Roave\BetterReflection\Reflection\Reflection;
 use RuntimeException;
+use function sprintf;
```

Comment:
Is it really needed?
Tests passes without this.

Reply:
Gonna likely propose the adoption of `doctrine/coding-standard` here, which enforces all functions to be imported to avoid static analysis issues, as well as runtime function lookups.

Reply:
Hmm, just seen that there already is an open pull request for that.

Reply:
Ok thanks :)
It's interesting point of view.

For `to avoid static analysis issues` is it not a problem from static analysis?
And for `function lookups` I don't understand where is the problem with that.
Have you an example? To help me to understand where it could be a problem.

Reply:
if you call a function `foo()` you have no idea if it's a function declared in the same namespace or a global function. PHP falls back on the global scope for functions and constants whereas you never have the ambiguity with classes: you either need to use the FQCN e.g. `\Closure` or import the symbol with a use statement `use Closure`

Reply:
> For `to avoid static analysis issues` is it not a problem from static analysis?

an example I have is this one:

```php
namespace Potato;

define('POTATO', 'POTATO');
```

Is `Potato::POTATO` defined here? Or is `define()` a user-defined function?

You can only be sure of it if it was written this way: 


```php
namespace Potato;

use function define;

define('POTATO', 'POTATO');
```

I hope that clears it up.